boa3/builtin/interop/runtime/notification.py

```diff
@@ -4,6 +4,17 @@
 
 
 class Notification:
+    """
+    Represents a notification.
+
+    :var script_hash: the script hash of the notification sender
+    :vartype script_hash: UInt160
+    :var event_name: the notification's name
+    :vartype event_name: str
+    :var state: the object representing the notification content, which can be of any type such as value, string, array,
```

Comment:
I don't know how this is represented on the boa side, but on the C# side `state` is an VM type Array ([this](https://github.com/neo-project/neo-vm/blob/master/src/neo-vm/Types/Array.cs)) holding other VM types.
Maybe @meevee98 can have a look at this description

Reply:
On boa it's represented as a `tuple` (as specified in the following line)

```suggestion
    :var state: a tuple value storing all the notification contents.
```